highlevel/util/command_line_parser.py

```diff
@@ -0,0 +1,52 @@
+"""Code OUTech
```

Comment:
This file is not "Code OUTech Highlevel".
Shouldn't this give information about the current file ?